### DISCUSSION TITLE
Rootless podman 5 problem?

### DISCUSSION THREAD

**Discussion by Tokariew** [wg-easy/wg-easy#1421] [Q&A]
after updating podman to version 5.0 which use pasta as default network driver and not slirp4netns i can't get rootless podman to work. I can connect with wg-easy but I can't access any site.
Someone had any luck running wg-easy with that config?

**Comment by kaaax0815** [comment]
Duplicate of https://github.com/wg-easy/wg-easy/issues/1422
